rackcity/views/rack_views.py

```diff
@@ -1,12 +1,20 @@
 from django.core.exceptions import ObjectDoesNotExist
 from django.http import HttpResponse, JsonResponse
 from rackcity.models import Rack, ITInstance
-from rackcity.api.serializers import RackSerializer, ITInstanceSerializer, RecursiveITInstanceSerializer
+from rackcity.api.serializers import RackSerializer, ITInstanceSerializer
 from rackcity.api.objects import RackRangeSerializer
 from rest_framework.decorators import permission_classes, api_view
 from rest_framework.permissions import IsAuthenticated, IsAdminUser
 from http import HTTPStatus
-
+from rackcity.views.rackcity_utils import get_rack_detailed_response
+@api_view(['GET'])
```

Comment:
this is incredibly picky but should add 2 empty lines here for linting 🤪

Reply:
we need AUTO LINTING